transcom/mymove
# B-21977 INT - Services Counselor can select counseling office when creating a move

## [B-21977](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21977)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a services counselor
3. Navigate to the customer search page
4. Search for any customer, maybe Spacemen, Leo
5. Create a new move for you selected user
6. When creating orders for your new move ensure the counseling office dropdown 
7. Select a counseling office and complete the orders form (for ease make sure it's origin in the same GBLOC as the services counselor you're using)
8. Add any shipment to your newly created move so it will show up on your queue
9. Navigate to your counseling queue, find the move you just created, and ensure the list counseling office matches the office you selected in step 7. 

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
